db/seeds.rb

```diff
@@ -116,18 +170,59 @@ def child_named(full_name, min_birthday: MIN_BIRTHDAY,
 # Subsidy Rules
 # ---------------------------------------------
 
-rule_effective_date = Faker::Date.between(from: 10.years.ago, to: Time.zone.today)
+today = Date.current
 
 il_sr_rule = IllinoisSubsidyRule.first_or_create!
 
-sr_rule_1 = SubsidyRule.first_or_create!(
-  name: 'Rule 1',
+il_rules_start = today - 200
```

Comment:
Let's name this variable something clearer since we're using it so much below. `two_hundred_days_ago`?